foam_tokenwrapper/tests/pitzDaily/0/R

```diff
@@ -0,0 +1,52 @@
+/*--------------------------------*- C++ -*----------------------------------*\
```

Comment:
The rules of thumb for repositories is:
- Data that cannot be found in other repositories are commonly placed in the same repository or placed on a separate public repository since they are not necessary for the use of the code 
- For data that can be found in other repositories a set of instructions are needed so that they can be downloaded.

These rules of thumb try to make the repository manageable  and easy to use. They also avoid issues with licencing since data have their own license and thus distribution in the same repository might cause issues.